components/support/nimbus-fml/src/backends/swift/gen_structs/mod.rs

```diff
@@ -7,15 +7,30 @@ use std::collections::HashSet;
 use crate::{
     backends::{CodeDeclaration, CodeOracle, CodeType, TypeIdentifier},
     intermediate_representation::{FeatureDef, FeatureManifest, TypeFinder},
+    parser::AboutBlock,
 };
 mod bundled;
 mod common;
 mod enum_;
 mod feature;
 mod filters;
+mod imports;
 mod object;
 mod primitives;
 mod structural;
+
+impl AboutBlock {
+    fn nimbus_object_name_swift(&self) -> String {
+        let specific = self.swift_about.as_ref().unwrap();
```

Comment:
Hmmm Should this be called `swift_about_block` or something like that? `specific` seems odd to me as a variable name 😅